### ISSUE TITLE
[Bug]: mo was killed by oom when running all tpch 10G queries for 3 times 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#7607]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):44f922071871a89ce50fddb69b0884a9653c8249
- Hardware parameters:64 core 250G
- OS type:
- Others:
```

### Actual Behavior

[Thu Jan 12 20:38:46 2023] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0-1,global_oom,task_memcg=/user.slice/user-1029.slice/session-1696.scope,task=mo-service,pid=33218,uid=1029
[Thu Jan 12 20:38:46 2023] Out of memory: Killed process 33218 (mo-service) total-vm:275249168kB, anon-rss:250620076kB, file-rss:0kB, shmem-rss:0kB, UID:1029 pgtables:521628kB oom_score_adj:0
[Thu Jan 12 20:38:48 2023] wireguard: wg_128: Sending keepalive packet to peer 21 (119.45.7.76:51820)
[Thu Jan 12 20:38:55 2023] oom_reaper: reaped process 33218 (mo-service), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
[Thu Jan 12 20:38:57 2023] wireguard: k8svpn: Sending keepalive packet to peer 20 (146.56.201.158:31544)
[Thu Jan 12 20:38:57 2023] wireguard: k8svpn: Sending keepalive packet to peer 20 (146.56.201.158:31544)
[Thu Jan 12 20:38:58 2023] wireguard: wg_128: Receiving keepalive packet from peer 21 (119.45.7.76:51820)
[Thu Jan 12 20:39:07 2023] wireguard: k8svpn: Sending keepalive packet to peer 20 (146.56.201.158:31544)
[Thu Jan 12 20:39:13 2023] wireguard: wg_128: Sending handshake initiation to peer 21 (119.45.7.76:51820)
[Thu Jan 12 20:39:13 2023] wireguard: wg_128: Receiving handshake response from peer 21 (119.45.7.76:51820)

**Comment by nnsgmsone** [comment]
I will take a look

**Comment by nnsgmsone** [comment]
No progress at the moment

**Comment by nnsgmsone** [comment]
This problem should be caused by a memory leak, and is trying to fix it (more trouble).

**Comment by nnsgmsone** [comment]
fixed by #7661

**Comment by aressu1985** [comment]
tracking for a more day

**Comment by aressu1985** [comment]
tracking for a more day

**Comment by aressu1985** [comment]
fixed:ce1aae5e38bfab63d644440fb8113c455f31e975
